roles/kubernetes/control-plane/tasks/0010-etcd-link.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create etcd cert directory
+  ansible.builtin.file:
+    path: "{{ etcd_cert_dir }}"
+    state: directory
+    mode: '0750'
+  when:
+    - inventory_hostname  in groups['kube_control_plane']
+
+- name: Generate symlink to etcd certs
+  ansible.builtin.file:
+    src: "{{ etcd_cert_paths.client[item.src] }}"
+    dest: "{{ etcd_cert_dir }}/{{ item.dest }}"
```

Comment:
I think this is the other way around:
from `ansible-doc file`

```
- name: Create a symbolic link
  ansible.builtin.file:
    src: /file/to/link/to
    dest: /path/to/symlink
    owner: foo
    group: foo
    state: link
 ```

Reply:
yes all good then
if you replace the var for "cert" it will be:
(item.src = "cert"):
src: "{{ etcd_cert_paths.client[item.src] }}" -> " {{ etcd_cert_dir }}/node-{{ inventory_hostname }}.pem
dest: "{{ etcd_cert_dir }}/{{ item.dest }}" ->   {{ kube_etcd_cert_file }} -> kube-client-cert.pem

Reply:
Right... I must be tired :/ sorry.